src/util/Util.js

```diff
@@ -420,6 +420,10 @@ class Util {
    * - `DARK_GREY`
    * - `LIGHT_GREY`
    * - `DARK_NAVY`
+   * - `BLURPLE`
+   * - `GREYPLE`
+   * - `DARK_BUT_NOT_BLACK`
+   * - `NOT_QUITE_BLACK`
```

Comment:
these should also be added to the `ColorResolvable` type in the typings

Reply:
Fixed!